### PR TITLE
Node pm2 recipe

### PR DESCRIPTION
**Pull request by ctrombley**

### CODE REVIEW

File: test/manual/definitions/apm/node/node-linux2.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   "services": [
     {
       "id": "nodetron",
-      "source_repository": "https://github.com/newrelic/demo-nodetron.git",
+      "source_repository": "-b replace_supervisord_with_pm2 https://github.com/newrelic/demo-nodetron.git",
```
Comment (gabeobrien) [inline test/manual/definitions/apm/node/node-linux2.json:20]: The branch 'replace_supervisord_with_pm2' doesn't exist anymore.  We should pull from head, even though head start node under Supervisord.

Comment (ctrombley) [reply test/manual/definitions/apm/node/node-linux2.json]: 👍

**Review by michaelkro — APPROVED** [review]
lgtm :tada:

**Review by gabeobrien — APPROVED** [review]
+1 looks good to me!

**Pull request merged**